jdaviz/configs/specviz/plugins/parsers.py

```diff
@@ -9,15 +10,15 @@
 
 from jdaviz.core.events import SnackbarMessage
 from jdaviz.core.registries import data_parser_registry
-from jdaviz.utils import standardize_metadata
+from jdaviz.utils import standardize_metadata, download_uri_to_path
 
 
 __all__ = ["specviz_spectrum1d_parser"]
 
 
 @data_parser_registry("specviz-spectrum1d-parser")
 def specviz_spectrum1d_parser(app, data, data_label=None, format=None, show_in_viewer=True,
-                              concat_by_file=False):
+                              concat_by_file=False, cache=None, local_path=os.curdir, timeout=None):
```

Comment:
Why is this default different from the others?
```suggestion
                              concat_by_file=False, cache=None, local_path=None, timeout=None):
```